data/resources/plugins/renameitem.py

```
@@ -61,7 +67,49 @@ def callback(self, *args):
+
+    def on_answer_question_rename(self, dialog, response, data=tuple):
+        rename_widget, parent_dialog = data
+        srvdlg = self.app.get_service('dialogs')
+        if response == 'apply':
+            bsource = rename_widget.get_filepath_source()
+            source = os.path.join(self.repository.docs, bsource)
+            btarget = rename_widget.get_filepath_target()
+            target = os.path.join(self.repository.docs, btarget)
+            renamed = self.util.filename_rename(source, target)
+            if not renamed:
+                text = f"<big>Another document with the same name already exists in this repository.</big>"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)